regulations/templates/regulations/chrome.html

```diff
@@ -39,24 +39,43 @@
     {% block sub-head-drawer-icons %}
     <div class="toc-head">
         {% block sidebar-drawer-icons %}
-        <a href="#" id="panel-link" class="toc-toggle panel-slide">
-          <img src="{%static "regulations/img/open-caret.svg" %}" class="drawer-toggle-icon" alt="Open navigation drawer" />
-          <img src="{%static "regulations/img/close-caret.svg" %}" class="drawer-toggle-icon" alt="Close navigation drawer"/>
+        <a href="#" id="panel-link" class="toc-toggle panel-slide"
```

Comment:
in changing these to SVG the toggles seem to no longer be vertically aligned (or maybe I just never noticed?!) 

to adjust for SVG: 

```
.panel-slide {
   font-size: 1.2em;
}
```